contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -505,26 +507,37 @@ def get_queryset(self):
 
     def annotate_queryset(self, queryset):
         queryset = super(AdminChannelViewSet, self).annotate_queryset(queryset)
-
-        editor_query = (
```

Comment:
Doesn't surprise me that these are slow - I had to remove these from the original ChannelViewset implementation for slowing everything down on develop.